nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/java/org/apache/nifi/reporting/AbstractSiteToSiteReportingTask.java

```diff
@@ -73,25 +80,65 @@
             .build();
     static final PropertyDescriptor COMPRESS = new PropertyDescriptor.Builder()
             .name("Compress Events")
+            .displayName("Compress Events")
             .description("Indicates whether or not to compress the data being sent.")
             .required(true)
             .allowableValues("true", "false")
             .defaultValue("true")
             .build();
     static final PropertyDescriptor TIMEOUT = new PropertyDescriptor.Builder()
             .name("Communications Timeout")
+            .displayName("Communications Timeout")
             .description("Specifies how long to wait to a response from the destination before deciding that an error has occurred and canceling the transaction")
             .required(true)
             .defaultValue("30 secs")
             .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
             .build();
     static final PropertyDescriptor BATCH_SIZE = new PropertyDescriptor.Builder()
             .name("Batch Size")
+            .displayName("Batch Size")
             .description("Specifies how many records to send in a single batch, at most.")
             .required(true)
             .defaultValue("1000")
             .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
             .build();
+    static final PropertyDescriptor TRANSPORT_PROTOCOL = new PropertyDescriptor.Builder()
```

Comment:
If I am not missing anything, these newly added PropertyDescriptors should be used by sub classes such as SiteToSiteBulletinReportingTask?